src/pc/gfx/gfx_pc.c

```diff
@@ -760,19 +764,43 @@ static void gfx_sp_pop_matrix(uint32_t count) {
 }
 
 static float gfx_adjust_x_for_aspect_ratio(float x) {
-    return x * (4.0f / 3.0f) / ((float)gfx_current_dimensions.width / (float)gfx_current_dimensions.height);
+    return x * gfx_current_dimensions.x_adjust_ratio;
 }
 
 static void OPTIMIZE_O3 gfx_sp_vertex(size_t n_vertices, size_t dest_index, const Vtx *vertices, bool luaVertexColor) {
+    float globalLightCached[2][3] = {
```

Comment:
`globalLightCached` is only used within a condition, may as well only do this math inside that condition.